easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,422 @@
+        'checksums': ['29872e92839765e546828bb7754a68c418d927cd064fd4708fab9fe9c8bb116b'],
+    }),
+    ('Jinja2', '2.11.3', {
+        'checksums': ['a6d58433de0ae800347cab1fa3043cebbabe8baa9d29e668f1c768cb87a333c6'],
+    }),
+    ('packaging', '20.9', {
+        'checksums': ['5b327ac1320dc863dca72f4514ecc086f31186744b84a230374cc1fd776feae5'],
+    }),
+    ('sphinxcontrib-websupport', '1.1.2', {
+        'modulename': 'sphinxcontrib.websupport',
+        'checksums': ['1501befb0fdf1d1c29a800fdbf4ef5dc5369377300ddbdd16d2cd40e54c6eefc'],
+    }),
+    ('Pygments', '2.5.2', {
+        'checksums': ['98c8aa5a9f778fcd1026a17361ddaf7330d1b7c62ae97c3bb0ae73e0b9b6b0fe'],
+    }),
+    ('imagesize', '1.3.0', {
+        'checksums': ['cd1750d452385ca327479d45b64d9c7729ecf0b3969a58148298c77092261f9d'],
+    }),
+    ('docutils', '0.17.1', {
+        'checksums': ['686577d2e4c32380bb50cbb22f575ed742d58168cee37e99117a854bcd88f125'],
+    }),
+    ('snowballstemmer', '2.2.0', {
+        'checksums': ['09b16deb8547d3412ad7b590689584cd0fe25ec8db3be37788be3810cbf19cb1'],
+    }),
+    ('Babel', '2.9.1', {
+        'checksums': ['bc0c176f9f6a994582230df350aa6e05ba2ebe4b3ac317eab29d9be5d2768da0'],
+    }),
+    ('alabaster', '0.7.12', {
+        'checksums': ['a661d72d58e6ea8a57f7a86e37d86716863ee5e92788398526d58b26a4e4dc02'],
+    }),
+    ('typing', '3.10.0.0', {
+        'checksums': ['13b4ad211f54ddbf93e5901a9967b1e07720c1d1b78d596ac6a439641aa1b130'],
+    }),
+    ('Sphinx', '1.8.6', {
+        'checksums': ['e096b1b369dbb0fcb95a31ba8c9e1ae98c588e601f08eada032248e1696de4b1'],
+    }),
+    ('sphinx-bootstrap-theme', '0.8.0', {
+        'checksums': ['038ee7e89478e064b5dd7e614de6f3f4cec81d9f9efbebb06e105693d6a50924'],
+    }),
+    ('colorama', '0.4.4', {
+        'checksums': ['5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b'],
+    }),
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
+    ('psutil', '5.9.0', {
+        'checksums': ['869842dbd66bb80c3217158e629d6fceaecc3a3166d3d1faee515b05dd26ca25'],
+    }),
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('singledispatch', '3.7.0', {
+        'checksums': ['c1a4d5c1da310c3fd8fccfb8d4e1cb7df076148fd5d858a819e37fffe44f3092'],
+    }),
+    ('sortedcontainers', '2.4.0', {
+        'checksums': ['25caa5a06cc30b6b83d11423433f65d1f9d76c4c6a0c90e3379eaa43b9bfdb88'],
+    }),
+    ('intervaltree', '3.1.0', {
+        'checksums': ['902b1b88936918f9b2a19e0e5eb7ccb430ae45cde4f39ea4b36932920d33952d'],
+    }),
+    ('functools32', '3.2.3-2', {
+        'checksums': ['f6253dfbe0538ad2e387bd8fdfd9293c925d63553f5813c4e587745416501e6d'],
+    }),
+    ('futures', '3.3.0', {
+        'modulename': False,
+        'checksums': ['7e033af76a5e35f58e56da7a91e687706faf4e7bdfb2cbc3f2cca6b9bcda9794'],
+    }),
+    ('glob2', '0.6', {
```

Comment:
are you sure 0.7 won't work?

Reply:
Yes, won't work, glob <= 0.6 is required by one of the extensions below